iModelCore/ECPresentation/Source/Content/ContentDescriptorBuilder.cpp

```diff
@@ -1236,8 +1270,13 @@ struct ContentDescriptorBuilderImpl : ContentSpecificationsHandler
 
         if (nullptr != m_specification && ShouldCreateFields(*m_descriptor))
             {
-            int count = AddCalculatedFields(m_specification->GetCalculatedProperties(), nullptr);
-            DIAGNOSTICS_LOG(DiagnosticsCategory::Content, LOG_TRACE, LOG_INFO, Utf8PrintfString("Added %" PRIu64 " fields from specification.", (uint64_t)count));
+            static const RelatedClassPath s_emptyPath;
+            static const std::unordered_set<ECClassCP> s_emptyClassesSet;
+            bvector<RelatedPropertiesSpecification const*> specsStack;
+            PropertyAppenderPtr appender = _CreatePropertyAppender(s_emptyClassesSet, s_emptyPath, nullptr, specsStack, nullptr);
+            size_t count = 0;

```

Comment:
won't the names conflict if we start at 0 both when adding spec calculated properties and when adding them from modifiers?

Reply:
No, because this `AddCalculatedFields` will always add calculated properties, no matter previously added properties from content modifiers. If there would be already added property with a name like `CalculatedProperty_0`, the new unique name would be generated: `CalculatedProperty_0_0`,